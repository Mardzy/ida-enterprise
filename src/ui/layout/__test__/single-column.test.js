import SingleColumn from "../single-column";

describe("The single column component", () => {

  it("renders its content", () => {
    const content = () => <div className="content">Content</div>;

    const wrapper = mount(
      <SingleColumn content={content}/>
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });

});