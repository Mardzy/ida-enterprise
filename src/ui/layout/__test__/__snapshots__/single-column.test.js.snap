// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The single column component renders its content 1`] = `
<SingleColumn
  content={[Function]}
>
  <Grid
    columns={1}
    stackable={true}
  >
    <div
      className="ui stackable one column grid"
    >
      <GridColumn>
        <div
          className="column"
        >
          <div
            className="content"
          >
            Content
          </div>
        </div>
      </GridColumn>
    </div>
  </Grid>
</SingleColumn>
`;
